.github/workflows/cd.yml

```diff
@@ -0,0 +1,22 @@
+name: CD
+on:
+  push:
+    tags:
+      - v*  
+jobs:
+  cd:
+    name: CD
+    runs-on: ubuntu-latest
+    environment: CD
+    steps:
+      - uses: actions/setup-node@v3
+        with:
+          registry-url: "https://registry.npmjs.org"
+          node-version-file: '.nvmrc'
+          cache: 'npm'
+      - run: npm ci
+      - run: npm run build
+      - run: npm publish
```

Comment:
It would have run on CI prior, so I reckon this is not necessary.
After all, the tag is not a new commit but a new ref on an existing commit

Reply:
Wouldn't the builds run in parallel instead of in series though? Or is there a condition that ensures the `CD` workflow only runs after a successful `CI` workflow?

Reply:
There's no protection indeed.
Eventually, it could be interesting to automate the release when the commit reach the main branch (out-of-scope for now, I want to KISS)

Reply:
```suggestion
      - run: npm test
      - run: npm publish
```